examples/headsail-bsp/src/tb.rs

```diff
@@ -4,6 +4,7 @@ use crate::uart::uart_write;
 
 pub const TAG_FAIL: &str = "[FAIL]";
 pub const TAG_PASS: &str = "[PASS]";
+pub const TAG_OK: &str = "[OK]";
```

Comment:
Please document this with a `///`. Make sure the difference between [OK] and [PASS] is clear.